app/commands/git/temp/import_pull_requests.rb

```diff
@@ -0,0 +1,32 @@
+module Git
+  module Temp
+    class ImportPullRequests
+      include Mandate
+
+      def call
+        repos.each do |repo|
+          ImportPullRequestsForRepo.(repo)
+        rescue StandardError => e
+          Rails.logger.error "Error importing pull requests for #{repo}: #{e}"
+        end
+      end
+
+      private
+      def repos
+        fetched = []
+
+        loop.with_index do |_, page|
+          response = octokit_client.search_repositories("org:exercism is:public", page: page + 1, per_page: 100)
```

Comment:
I could have used the graph API here too, which would probably be slightly more efficient as it would return far less data. The code would become a lot less readable though, mainly due to the way paging works in the graphql API. So I opted for using the regular API now.

Reply:
Probably worth adding this as a code comment. And into other places where this sort of decision has been made.